### DISCUSSION TITLE
How do i deactivate Other Servers?

### DISCUSSION THREAD

**Discussion by MConsuela** [mastodon/mastodon#28372] [Q&A]
Hello

> click on live feeds
> click on other servers
> see content from servers that i or my users follow

problem is the "other servers" tab shows stuff that i may personally watch or download, buuuut which "sharing" is illegal in my country, so i might want to deactivate the "other server" Tab, any clue how i can do that? 

Thanks.

**Comment by vmstan** [comment]
This is currently not possible as a built-in option.

**Comment by cooperaj** [comment]
Given the introduction of the OAS in the UK could this now be considered as a bit of a priority? Reducing the surface area for user generated content by blocking this endpoint would greatly reduce the amount of assessed risk that we're all going to have to be doing here (even for single user instances).

Ben Tasker ([@bentasker](https://github.com/bentasker)) put together a [really good article](https://www.bentasker.co.uk/posts/blog/security/restricting-unauthenticated-access-to-mastodons-public-feeds.html) on how we might go about making these endpoints authenticated only (but not all of them importantly) but I'd much rather this was something configurable.

EDIT.
Found a relevant and still open issue https://github.com/mastodon/mastodon/issues/22620
